index.js

```diff
@@ -37,6 +39,17 @@ try {
 
     const vswhereResult = spawn(vswherePath, vswhereArgs, {encoding: 'utf8'})
     if (vswhereResult.error) throw vswhereResult.error
+
+    if (verbose) {
+      const args = [
+        '-nologo',
+        '-latest',
+        '-products', '*',
+      ].concat(requiresArg)
+      const details = spawn(vswherePath, args, { encoding: 'utf8' })
```

Comment:
Perhaps just set the `stdio` args option to `[ 'ignore', 'inherit', 'inherit' ]` and then not bother with `details` ? That'll be more efficient (no buffering of input just to dump it back out to standard output), though that probably doesn't matter in any real way here.